### PR TITLE
Update Anaconda WebUI plugin to reflect changes made in anaconda repo

### PR DESCRIPTION
**Pull request by velezd**

### CODE REVIEW

**Review by jstodola — DISMISSED** [review]
It looks good to me.

File: doc/workflows/anaconda-webui.rst [hunk 1]
```diff
@@ -50,12 +50,12 @@ Examples can be found in `default settings file <https://github.com/rhinstaller/
...
 AnacondaWebUI
 ^^^^^^^^^^^^^
-- **anaconda_repo** - URL to Anaconda git repository or path to local directory (use file://)
-- **cockpit_repo** - URL to Cockpit git repository, if anaconda_repo is local directory and
-  ``anaconda/ui/webui/test/common`` exists, this repo won't be used.
+- **webui_repo** - URL to anaconda-webui git repository or path to local directory (use file://)
+- **cockpit_repo** - URL to Cockpit git repository, if webui_repo is local directory and
+  ``./test/common`` exists, this repo won't be used.
 - **cockpit_branch** - Cockpit git branch
 - **bots_repo** - URL to Cockpit Bots git repository, if anaconda_repo is local directory and
```
Comment (jstodola) [inline doc/workflows/anaconda-webui.rst]: "anaconda_repo" should be replaced with "webui_repo"

File: libpermian/plugins/anaconda_webui/commands.py [hunk 2]
```diff
@@ -64,11 +64,11 @@ def run_awebui_tc_command(base_parser, args):
 
     os.environ.setdefault(
         "PIPELINE_library_directPath",
-        os.path.join(options.path_to_anaconda, 'ui/webui/test/end2end/'),
+        os.path.join(options.path_to_webui, 'test/test_cases/'),
     )
     os.environ.setdefault(
         "PIPELINE_AnacondaWebUI_anaconda_repo",
```
Comment (jstodola) [inline libpermian/plugins/anaconda_webui/commands.py]: Should this be changed to `"PIPELINE_AnacondaWebUI_webui_repo"`?

Comment (velezd) [reply libpermian/plugins/anaconda_webui/commands.py]: Thanks

File: doc/workflows/anaconda-webui.rst (hunk 1 above)
Comment (velezd) [reply doc/workflows/anaconda-webui.rst]: It is, or am I missing something?

**Review by jstodola — APPROVED** [review]
It looks good to me.

**Pull request merged**